scripts/create-indexes.sh

```diff
@@ -16,16 +16,12 @@
 
 set -e
 
-if [[ $# -lt 3 ]]; then
-  echo "usage: create-indexes.sh <credentials> <project> <indexes_file>" 1>&2
+if [[ $# -lt 2 ]]; then
+  echo "usage: create-indexes.sh <project> <indexes_file>" 1>&2
   exit 1
 fi
 
-CREDENTIALS=$1
-PROJECT=$2
-INDEXES_FILE=$3
+PROJECT=$1
+INDEXES_FILE=$2
 
-export CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE="$CREDENTIALS"
```

Comment:
`CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE` still needs to be set for `gcloud`, similiar to how it is done in [Kubernetes Engine](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/scripts/wait-for-cluster.sh#L18-L20).

Reply:
What is it used for?